stories/__tests__/SQFormDropdown.stories.test.js

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import * as Yup from 'yup';
+import {render, screen, within} from '@testing-library/react';
+import {composeStories} from '@storybook/testing-react';
+import userEvent from '@testing-library/user-event';
+import * as stories from '../SQFormDropdown.stories';
+
+const {SQFormDropdown} = composeStories(stories);
+
+it('should render list of options', () => {
+  render(<SQFormDropdown size="auto" />);
+
+  const expandButton = screen.getByRole('button', {name: /state/i});
+
+  userEvent.click(expandButton);
+
+  const optionsList = screen.getByRole('listbox');
+  expect(optionsList).toBeInTheDocument();
+  expect(within(optionsList).getAllByRole('option')).toHaveLength(3);
+});
+
+it('should render with empty initial value', () => {
+  render(<SQFormDropdown size="auto" />);
+
+  const labelValue = screen.getByRole('button', {name: /state/i});
+  expect(labelValue).toHaveTextContent('- -');
+});
+
+it('should render with non-empty initial value', () => {
+  render(
+    <SQFormDropdown size="auto" SQFormProps={{initialValues: {state: 'MO'}}} />
```

Comment:
nitpick: it _may_ be useful to export the `MOCK_STATE_OPTIONS` from the dropdown story so it can be used in the test, that might give a bit more context than 'MO' and 'Missouri'. Not a super necessary change though